src/py/flwr/server/app.py

```diff
@@ -438,17 +504,45 @@ def _add_args_common(parser: argparse.ArgumentParser) -> None:
     )
 
 
-def _add_arg_driver_api_address(parser: argparse.ArgumentParser) -> None:
+def _add_args_driver_api(parser: argparse.ArgumentParser) -> None:
     parser.add_argument(
         "--driver-api-address",
         help=f"Driver API gRPC server address. Default: {ADDRESS_DRIVER_API}",
         default=ADDRESS_DRIVER_API,
     )
 
 
-def _add_arg_fleet_api_address(parser: argparse.ArgumentParser) -> None:
-    parser.add_argument(
-        "--fleet-api-address",
-        help=f"Fleet API gRPC server address. Default: {ADDRESS_FLEET_API_GRPC}",
-        default=ADDRESS_FLEET_API_GRPC,
+def _add_args_fleet_api(parser: argparse.ArgumentParser) -> None:
+    # Fleet API transport layer type
+    ex_group = parser.add_mutually_exclusive_group()
+    ex_group.add_argument(
+        "--grpc",
+        action="store_const",
+        dest="fleet_api_type",
+        const="grpc",
+        default="grpc",
+        help="Start a gRPC-based Fleet API server",
+    )
+    ex_group.add_argument(
+        "--rest",
```

Comment:
Should we name this also `--use-rest`?

Reply:
We could, but then we'd have to also rename `--grpc` to `--use-grpc`

Reply:
Thinking about it, I'd prefer not to. We'd then also have to rename `--rest-fleet-api-address` to `--use-rest-fleet-api-address`

Reply:
We could rename the `start_client` argument `use_rest` to `rest` though

Reply:
Either way is fine for me. I am going to approve you can change it if you want to. I can reapprove afterwards if you do.